event_model/__init__.py

```diff
@@ -896,6 +898,51 @@ def verify_filled(event_page):
                                    f"`event_model.Filler`.")
 
 
+def sanitize_np(doc):
+    '''Return a copy with any numpy objects converted to built-in Python types.
+    This is a faster version of sanitize_doc which only converts numpy objects.
+
+    This function takes in an event-model document and returns a copy with any
+    numpy objects converted to built-in Python types. It is useful for
+    sanitizing documents prior to sending to any consumer that does not
+    recognize numpy types, such as a MongoDB database or a JSON encoder.
+
+    Parameters
+    ----------
+    doc : dict
+        The event-model document to be sanitized
+
+    Returns
+    -------
+    sanitized_doc : event-model document
+        The event-model document with numpy objects converted to built-in
+        Python types.
+    '''
+
+    def iterate_sanitize(doc):
+        if hasattr(doc, 'items'):
+            for value in doc.values():
+                iterate_sanitize(value)
+        elif isinstance(doc, collections.abc.Iterable) and not isinstance(str):
+            doc = list(doc)  # Change tuples to lists
+            for value in doc:
+                iterate_sanitize(value)
+        else:
+            doc = sanitize_item(doc)
+
+    iterate_sanitize(doc)
```

Comment:
```suggestion
    return iterate_sanitize(doc)
```

Is this inner function even necessary? Seems like this outer function (i.e., `sanitize_np`) should be recursive

Reply:
I think my last commit addressed this point. Do you agree?